site/content/docs/5.0/migration.md

```diff
@@ -23,6 +23,7 @@ toc: true
 - Restored `offset` option for Dropdown, Popover and Tooltip plugins.
 - The default value for the `fallbackPlacements` is changed to `['top', 'right', 'bottom', 'left']` for better placement of popper elements.
 - All the events for the dropdown are now triggered on the dropdown toggle button and then bubbled up to the parent element.
+- Dropdown menus now have a `data-bs-popper="static"` attribute set when the trigger element has `data-bs-display="static"`. This is added by our JavaScript and helps us use custom position styles without interferring with Popper's positioning.
```

Comment:
`data-bs-popper="static"` is added whenever the `applyStyles` modifier (Popper's native config) is disabled. (Via data attributes, JavaScript or `popperConfig`)
`data-bs-popper="none"` is also added when the dropdown is inside the navbar and not using Popper. So that styles can be applied when the dropdown is not placed by Popper.

Reply:
@rohit2sharma95 you mean we should reword this? If so feel free suggest push a separate commit

Reply:
```suggestion
- Dropdown menus now have a `data-bs-popper="static"` attribute set when the trigger element has `data-bs-display="static"`. This is added by our JavaScript and helps us use custom position styles without interferring with Popper's positioning.
+ Dropdown menus now have a `data-bs-popper="static"` attribute set when the positioning of the dropdown is static and `data-bs-popper="none"` when dropdown is in the navbar. This is added by our JavaScript and helps us use custom position styles without interfering with Popper's positioning.
```